src/azul/plugins/metadata/anvil/indexer/transform.py

```diff
@@ -428,6 +431,9 @@ def _activity_polymorphic_types(self) -> AbstractSet[str]:
             if table['name'].endswith('activity')
         }
 
+    def _is_duos(self, dataset: JSON) -> bool:
```

Comment:
What motivated the moving of this method? The current placement seems more suitable to me, as it is closer to other mentions of the key `duos_id`.

Reply:
I've performed a refactoring that avoids the need to relocate the method.